src/app/services/answer.py

```diff
@@ -0,0 +1,41 @@
+"""Answer service"""
+
+from app import DB
+from app.helper.decorators import transaction_decorator
+from app.models import Answer
+
+
+class AnswerService:
+    """ answer service class"""
+
+    @staticmethod
+    def get_by_id(answer_id):
+        """
+
+        :param pk:
+        :return: answer with id=pk or None
+
+        """
+        return Answer.query.get(answer_id)
```

Comment:
For debug it is better to write: 
answer = Answer.query.get(answer_id)
return answer